ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewGroup.java

```diff
@@ -455,6 +456,50 @@ private void updateSubviewClipStatus(View subview) {
     }
   }
 
+  @Override
+  public boolean getChildVisibleRect(View child, Rect r, android.graphics.Point offset) {
+    // This is based on the Android ViewGroup implementation, modified to clip child rects
+    // if overflow is set to ViewProps.HIDDEN. This effectively solves Issue #23870 which
+    // appears to have been introduced by FLAG_CLIP_CHILDREN being forced false
+    // regardless of whether clipping is desired.
+    final RectF rect = new RectF();
+    rect.set(r);
+
+    child.getMatrix().mapRect(rect);
+
+    final int dx = child.getLeft() - getScrollX();
+    int dy = child.getTop() - getScrollY();
```

Comment:
`final` this too? :)

Reply:
Oops. Must have been a copy-paste error (I was working across two repos).